api/USERS_GUIDE.md

```diff
@@ -117,6 +121,14 @@ curl -i -X GET "https://www.pgdp.org/api/v1/projects?genre=Other" \
     -H "X-API-KEY: $API_KEY"
 ```
 
+Search for all projects with genre `Science` or `Sports`:
+
+```bash
+curl -i -X GET "https://www.pgdp.org/api/v1/projects?genre[]=Science&genre[]=Sports" \
+    -H "Accept: application/json" \
+    -H "X-API-KEY: $API_KEY"
+```
+
```

Comment:
Should we clarify it even more by giving an example of OR and AND in the same query? e.g. monster in title and genre either Science or sports.

Reply:
Sure, that gives us a chance to give a pseudocode example of the AND/ORing too. Just added that.